substrate/network/src/message.rs

```diff
@@ -447,75 +253,12 @@ pub mod generic {
 		/// Remote method call response.
 		RemoteCallResponse(RemoteCallResponse),
 		/// Chain-specific message
+		#[codec(index = "15")]
```

Comment:
Shouldn't this be `255`?

Reply:
Great catch, don't know where I took 15 from.